compiler/src/iree/compiler/InputConversion/Common/Passes.td

```diff
@@ -21,6 +21,13 @@ def ImportMLProgram :
   let constructor = "mlir::iree_compiler::createImportMLProgramPass()";
 }
 
+def LinalgQuantizedConvToConvPass
+    : Pass<"iree-linalg-quantized-conv-to-conv", "func::FuncOp"> {
+  let summary = "lower quantized_conv to conv";
+  let constructor = "mlir::iree_compiler::createLinalgQuantizedConvToConvPass()";
+  // let dependentDialects = ["mlir::linalg::LinalgDialect"];
```

Comment:
This could be dropped (  I think this is linalg on input expected before it triggers)

Reply:
Done.